forte/data/extractor/base_extractor.py

```diff
@@ -257,17 +257,25 @@ def extract(self, pack: DataPack,
         """
         pass
 
-    def remove_from_pack(self, pack: DataPack,
+    def pre_evaluation_action(self, pack: DataPack,
                 instance: Annotation):
         r"""Functionality: Remove the existing feature of the instance
         in a pack. We might remove an attribute under an entry or remove
         the entry itself directly, which will depend on the different
-        type of extractors.
+        type of extractors. This function is called before the evaluation
```

Comment:
So `removal` won't be the main functionalities. You can simply say this function is called before evaluation. The users can use this chance to remove existing gold standards or do anything else. We don't assume that they will only do the deletion. We can suggest that one common thing to do is deletion.

Reply:
Got it.